recipe/meta.yaml

```diff
@@ -40,6 +40,7 @@ requirements:
     - six
     - beniget >=0.2.0
     - xsimd =7.2.2  # [not win]
+    - colorlog  # [py3k]
```

Comment:
colorlog not supported anymore on py27 ? not that it matters, just out of curiosity.